### ISSUE TITLE
First option button now has a blue halo around the white check part - looks like the interface is recommending a default answer 

### ISSUE THREAD

**Issue by mschwamb** [zooniverse/Panoptes-Front-End#2563]
The interface is now putting a blue halo around the white part of the button on check bock for the first response now for every question task. It makes it look like there's a default or preferred/recommended answer which is not true in my case that the first is typical especially for the multiple choice/multiple answer ones. Can that please be taken off, I think this will bias people's classifications with people thinking that the interface is hinting at the default or typical option

<img width="1117" alt="screen shot 2016-05-10 at 4 33 15 pm" src="https://cloud.githubusercontent.com/assets/753325/15168430/303c5c52-16cd-11e6-979d-519d67cc8567.png">
<img width="1125" alt="screen shot 2016-05-10 at 4 32 58 pm" src="https://cloud.githubusercontent.com/assets/753325/15168441/42dd85ac-16cd-11e6-888d-78e5ce908a2b.png">

<img width="1126" alt="screen shot 2016-05-10 at 4 33 06 pm" src="https://cloud.githubusercontent.com/assets/753325/15168433/37f5bb50-16cd-11e6-89a2-977558387056.png">

<img width="1139" alt="screen shot 2016-05-10 at 4 36 06 pm" src="https://cloud.githubusercontent.com/assets/753325/15168457/67a15d0a-16cd-11e6-9c85-ae7442a8d293.png">

only when you select a response does the blue outline in the white area go away for the first answer button

<img width="1128" alt="screen shot 2016-05-10 at 4 37 02 pm" src="https://cloud.githubusercontent.com/assets/753325/15168477/90b75424-16cd-11e6-99be-6c81bc55a844.png">

**Comment by mschwamb** [comment]
I see it on Firefox and Chrome. Haven't tried other things. Also when click done it now highlights the Next button emphasizing it more than the Talk button. 

<img width="1096" alt="screen shot 2016-05-10 at 5 05 29 pm" src="https://cloud.githubusercontent.com/assets/753325/15168918/968da8c2-16d1-11e6-989d-5f847870b238.png">

**Comment by aliburchard** [comment]
gah, would be good to get this fixed quickly - I'd be worried that this will bias users answers.

**Comment by alexbfree** [comment]
possibly related to https://github.com/zooniverse/Panoptes-Front-End/pull/2508 @simensta @eatyourgreens ?

**Comment by marten** [comment]
Sounds likely. But getting rid of the halo would negate the usability benefits of being able to use the keyboard to classify?

**Comment by mschwamb** [comment]
I'm really concerned the blue halo is strongly biasing my the current clicks and my subsequent results  from my project builder projects because it immediately  looks to me to be  a default or encouraged answer. I'd prefer to have the previous keyboard access that was had before this change was made.

**Comment by aliburchard** [comment]
what if we made the default resting spot for the halo (and keyboard selection starting point) at the last focus-able object before the answers (which happens to be the metadata button). So that a user hits the next arrow and then the first answer is selected.

**Comment by alexbfree** [comment]
@marten > i am not sure what is the convention here, i guess something has to have keyboard focus by default so maybe we need to have a hidden non-functional element hidden by default? or give focus to something more inconsequential like the [i] button.

Definitely agree we need to fix this. I bet Jim'll have an idea about the best way to approach this problem.

**Comment by alexbfree** [comment]
perhaps we should try rolling back #2508 temporarily and see if that fixes it/gets rid of the problem until we have a solution...

**Comment by alexbfree** [comment]
Oh! I commented before your comment Ali, seems like we both had the same idea of giving focus to the metadata button but for different reasons.

**Comment by alexbfree** [comment]
Had a quick Inspect on Comet Hunters, looks like it's the `active` css class that determines the current focus. another temporary workaround would be to disable any styling on that class so that the halo doesn't show up - but i think that rolling back or changing the default would be preferable.

**Comment by mschwamb** [comment]
If this for accessibility/ people that will use the keyboard than perhaps there can be a setting to turn this on or a way the browser knows a person needs this kind of blue halo and will know what to do with it, but since most people do not use the keyboard to classify I really worry about not being to use the data from these classifications in the long term. I'd appreciate a rollback or a turn off or some way to have it not show up if you're not using the keyboard to classify

**Comment by marten** [comment]
I'm hoping @eatyourgreens will soon chime in with his accessibility knowledge. I too think we should probably focus some element just before the first option, but the buttons under the subject are not ideal. A hidden element would mean there's no halo at all, which in turn means it's not obvious you can even easily use the keyboard.

Perhaps long-term it would be better to give all options hotkeys, or in the case of the survey task to allow drilling down by incremental search.

But it's been in there for 13 days now, waiting a couple of hours to gather more intel rather than hastily reverting probably won't hurt.

**Comment by eatyourgreens** [comment]
The blue ring is the OS X keyboard focus indicator. It will show up on Macs but not for people classifying on Windows. We've implemented autofocus so that people can classify with the keyboard (previously this was pretty much impossible and you were forced to use a mouse.) Whether or not it appears is controlled by the system settings (under System Preferences > Keyboard > Shortcuts I think.)

Our implementation is based on the one currently in use on Penguin Watch, Science Gossip and Orchid Observers. I'm not aware of it introducing a bias into the classifications that those projects have recorded, and they've had autofocus for keyboard access for over a year now.

@alexbfree I don't know the specifics of webkit's implementation of focus, but I imagine it uses the `:focus` pseudoclass. If we do remove it (with `outline: none`) then we should implement some sort of visual indicator of our own to indicate when an answer has focus, which will still make the first answer look different from the others.

**Comment by mschwamb** [comment]
@eatyourgreens for the projects you just mentioned they look like to me 

<img width="1219" alt="screen shot 2016-05-11 at 2 18 50 am" src="https://cloud.githubusercontent.com/assets/753325/15180714/f26aedd8-171e-11e6-9395-a816b5f1f884.png">

<img width="1206" alt="screen shot 2016-05-11 at 2 20 01 am" src="https://cloud.githubusercontent.com/assets/753325/15180713/f268ef4c-171e-11e6-83cf-e944516b5406.png">

Where I don't see a box or aura highlighting an answer If I tab a bit then I get this where it looks like it's highlighting for keyboard 

<img width="1203" alt="screen shot 2016-05-11 at 2 18 54 am" src="https://cloud.githubusercontent.com/assets/753325/15180712/f26781b6-171e-11e6-8e8c-f16f581c6798.png">

Can what is being done in science gossip on the first focus be done in panotpes so that the next button looks exactly like the Talk button be implemented for panoptes then?

**Comment by eatyourgreens** [comment]
@mschwamb I'm not sure I follow. Those buttons look like this, for me, on Science Gossip.
![sciencegossip-focus](https://cloud.githubusercontent.com/assets/59547/15181180/9dd6596a-177d-11e6-9b7b-f725571182b0.jpg)

**Comment by CKrawczyk** [comment]
@eatyourgreens it seems on Science Gossip you have to hit tab once before the focus lands on the answers, on PFE the focus is on the answers from the start.

**Comment by alexbfree** [comment]
@eatyourgreens I agree that we shouldn't hide the outlines. Was just brainstorming workarounds.

**Comment by alexbfree** [comment]
Yes I think @CKrawczyk has nailed the problem. Actually for me on Science Gossip. I have to hit tab 16 times before the radio button is keyboard-focussed. So definitely no chance of bias there.

**Comment by CKrawczyk** [comment]
@alexbfree I mean after hitting "next" tab has to be pressed once for the next question to have focus.

**Comment by alexbfree** [comment]
ah ok

**Comment by alexbfree** [comment]
@eatyourgreens are you still on this? or should we just revert for now?

**Comment by alexbfree** [comment]
Seems we can't just revert this, as we need a solution that will work for #2530 too. @simensta has a plan and is working on a new PR.

**Comment by eatyourgreens** [comment]
Yeah, readymade focuses the first form input for the first question in a decision tree, back button for the second and subsequent questions
https://github.com/zooniverse/zooniverse-readymade/blob/master/src/classify-page.coffee#L79-L82

**Comment by amyrebecca** [comment]
just want to point out that removing accessibility features also biases results, albeit in less-predictable ways.

**Comment by mschwamb** [comment]
The current solution works alleviates my major concerns. Thank you. Closing the ticket.

**Comment by eatyourgreens** [comment]
Downside of this is that now, on Comet Hunters, you have to press `tab` 25 times before you can answer each classification question, so we still need to find a way to make it easy to classify using the keyboard.
